src/api-reference/user-provisioning/v4.spend-user-provisioning.md

```diff
@@ -0,0 +1,702 @@
+                    "customData": [
+                        {
+                            "id": "custom1",
+                            "value": "testing"
+                        },
+                        {
+                            "id": "orgUnit2",
+                            "value": "testSquadAlpha"
+                        }
+                    ]
+                },
+                "urn:ietf:params:scim:schemas:extension:spend:2.0:Approver": {
+                    "request": [
+                        {
+                        "approver": {
+                            "employeeNumber": "requestApprover"
+                        },
+                        "primary": true
+                        }
+                    ],
+                },
+            }
+        }
+    ]
+}
+```
+
+## <a name="get-users-endpoint"></a> `GET /spend/v4/Users` endpoint for retrieving all spend users
+
+`spend/v4/Users` endpoint for retrieving all the spend users for a given company. The result is paginated and can be filtered using the params listed below.
+
+### Scopes
+
+- `spend.user.general.read`
+
+Refer to [Scope Usage](#scope-usage) for full details.
+
+### Request
+
+```shell
+GET https://www.us.api.concursolutions.com/spend/v4/Users
+```
+
+#### URI
+
+```shell
+GET /spend/v4/Users
+```
+
+#### Headers
+
+- `concur-correlationid` is a SAP Concur specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
+- `company-uuid` the company UUID that users are being retrieved for and that your passed certificate has authority within.
+
+### Request Example
+
+```shell
+GET https://www.us.api.concursolutions.com/provisioning/v4/Users?&startIndex=1&itemsPerPage=4&filter=urn:ietf:params:scim:schemas:extension:spend:2.0:User:country+eq+%22US%22
+```
+
+#### Parameters
+
+| Name           | Type     | Format                                                     | Description                                                                |
+| -------------- | -------- | ---------------------------------------------------------- | -------------------------------------------------------------------------- |
+| `startIndex`   | `int`    |                                                            | The starting index of the paginated result (indexes from 1)                |
+| `itemsPerPage` | `int`    |                                                            | The number of user resources to return on a single page (max: 100)         |
+| `filter`       | `string` | [RFC](https://tools.ietf.org/html/rfc7644#section-3.4.2.2) | The SCIM compliant filter string to be used when retrieving user resources |
+
+Note: Currently not all aspects of SCIM filtering are supported. As of right now the following fields are implemented with the corresponding set of operators:
+
+- `CashAdvanceAccountCode` { **eq**, **ne** }
```

Comment:
Do we support the field in any capitalization type or is UpperCamelCase required?